CONTRIBUTING.md

```diff
@@ -154,6 +153,36 @@ This implies that rule names:
 When re-implementing rules from other linters, this convention is given more importance than
 preserving the original rule name.
 
+#### Rule testing: fixtures and snapshots
```

Comment:
(Everything above is just drive-by formatting changes added by me after the initial PR.)